conference.js

```diff
@@ -2137,7 +2137,7 @@ export default {
             UIEvents.AUDIO_OUTPUT_DEVICE_CHANGED,
             audioOutputDeviceId => {
                 sendAnalytics(createDeviceChangedEvent('audio', 'output'));
-                setAudioOutputDeviceId(audioOutputDeviceId)
+                setAudioOutputDeviceId(audioOutputDeviceId, APP.store.dispatch)
```

Comment:
How did this work previously without the dispatch parameter?

Reply:
I think this device output used to not save to settings and instead relied on lib-jitsi-meet to store it. Once the preferred device moved to redux, saving the preferred device didn't work.